config/locales/en.yml

```
@@ -253,7 +253,7 @@ en:
       notes_per_week: "%{note_count} notes per week over this period."
       edits_per_week: "%{count} edits per week over this period."
       notes_wikis_comments: |
-          <b>%{notes_count} notes</b> and <b>%{wikis_count} wiki edits and</b>
+          <b>%{notes_count} notes</b> and <b>%{wikis_count} wiki edits</b>
```

```suggestion
          <b>%{notes_count} notes</b>, <b>%{wikis_count} wiki edits</b>
```
Hey @cesswairimu @publiclab/plots2-reviewers, how about changing "and" to ","?

---

We use notes_wikis_comments here only so it shouldn't be an issue too

---

Yeah sounds good @gauravano @Saad2714